src/Symfony/Component/Notifier/Bridge/Expo/ExpoTransport.php

```diff
@@ -91,10 +94,15 @@ protected function doSend(MessageInterface $message): SentMessage
             throw new TransportException('Unable to post the Expo message: '.$errorMessage, $response);
         }
 
-        $success = $response->toArray(false);
+        $result = $response->toArray(false);
+
+        $isError = 'error' === $result['data']['status'];
+        if ($isError) {
```

Comment:
```suggestion
        if ('error' === $result['data']['status']) {
```

No need for the one time variable

Reply:
is removed